components/console/esp_console.h

```diff
@@ -12,9 +12,7 @@ extern "C" {
 #include <stddef.h>
 #include "sdkconfig.h"
 #include "esp_err.h"
-
-// Forward declaration. Definition in linenoise/linenoise.h.
-typedef struct linenoiseCompletions linenoiseCompletions;
```

Comment:
Could we keep this as a forward declaration instead of pulling in the dependency on esp_linenoise.h header?